core/controllers/story_editor.py

```diff
@@ -59,60 +59,60 @@ def _require_valid_version(self, version_from_payload, story_version):
                 % (story_version, version_from_payload))
 
     @acl_decorators.can_edit_story
-    def get(self, story_id):
+    def get(self, storyId):
         """Populates the data on the individual story page."""
-        story = story_fetchers.get_story_by_id(story_id, strict=False)
-        topic_id = story.corresponding_topic_id
-        topic = topic_fetchers.get_topic_by_id(topic_id, strict=False)
-        skill_ids = topic.get_all_skill_ids()
```

Comment:
Oh, no, don't rename these variables. All variables local to the controller should still be in snake_case. Check my comment below.